### PR TITLE
Fix old math commands

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

There were still many references to `?` commands, which are now `%`.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/cmd_descs/cmd_descs.yaml [hunk 1]
```diff
@@ -112,10 +112,10 @@ commands:
             arg_str: ""
             comment: List defined macros
           - text: (
-            arg_str: foo; ?e Disassemble 10 bytes at 0x10000; pd 10 @ 0x10000)
+            arg_str: foo; echo Disassemble 10 bytes at 0x10000; pd 10 @ 0x10000)
```
Comment (XVilka) [inline librz/core/cmd_descs/cmd_descs.yaml:115]: Shouldn't it use quotes around `echo` argument?

Comment (XVilka) [reply librz/core/cmd_descs/cmd_descs.yaml:115]: Here and in other places in this PR

File: librz/core/tui/panels.c [hunk 2]
```diff
@@ -5272,7 +5272,7 @@ bool __handle_console(RzCore *core, RzPanel *panel, const int key) {
 			if (!strcmp(cmd, "clear")) {
 				rz_cons_clear00();
 			} else {
-				rz_core_cmdf(core, "?e %s %s>>$console", prompt, cmd);
+				rz_core_cmdf(core, "echo %s %s>>$console", prompt, cmd);
```
Comment (XVilka) [inline librz/core/tui/panels.c]: Can you use the API here? Should be straightforward, AFAIK.

**Review by XVilka — CHANGES_REQUESTED** [review]
Many broken tests, likely related to the offset changes:
```
[XX] db/cmd/cmd_pd_bytes pD -10
RZ_NOPLUGINS=1 /private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-1287038090/rizinorg/rizin/prefix/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'e asm.arch=x86
e asm.bits=64
e asm.bytes=true
wx 90909090909090909090 ; s 10 ; pD -10
' malloc://1024
-- stdout
--- expected
+++ actual
@@ -1,10 +1,10 @@
-            0x00000000      90             nop
-            0x00000001      90             nop
-            0x00000002      90             nop
-            0x00000003      90             nop
-            0x00000004      90             nop
-            0x00000005      90             nop
-            0x00000006      90             nop
-            0x00000007      90             nop
-            0x00000008      90             nop
-            0x00000009      90             nop
+            0x0000000a      90             nop
+            0x0000000b      90             nop
+            0x0000000c      90             nop
+            0x0000000d      90             nop
+            0x0000000e      90             nop
+            0x0000000f      90             nop
+            0x00000010      90             nop
+            0x00000011      90             nop
+            0x00000012      90             nop
+            0x00000013      90             nop

[**]                                  db/cmd/cmd_fuzzed    16173 OK      1011 BR       12 XX       34 FX
[XX] db/cmd/cmd_pd_bytes pD -10 @ 0
RZ_NOPLUGINS=1 /private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-1287038090/rizinorg/rizin/prefix/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'e asm.arch=x86
e asm.bits=64
e asm.bytes=true
e analysis.hasnext=0
wx b8010000004839ca7f00
pD -10 @ 10
' malloc://512
-- stdout
--- expected
+++ actual
@@ -1,3 +1,3 @@
-            0x00000000      b801000000     mov   eax, 1
-            0x00000005      4839ca         cmp   rdx, rcx
-        ,=< 0x00000008      7f00           jg    0xa
+            0x0000000a      b801000000     mov   eax, 1
+            0x0000000f      4839ca         cmp   rdx, rcx
+        ,=< 0x00000012      7f00           jg    0x14

[XX] db/cmd/cmd_pd_bytes pD -10 @ 10
RZ_NOPLUGINS=1 /private/var/folders/bc/yvvw2g2s4jlg9r8x7tf61xv800009d/T/woodpecker-local-1287038090/rizinorg/rizin/prefix/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'e asm.arch=x86
e asm.bits=64
e asm.bytes=true
e analysis.hasnext=0
wx b8010000004839ca7f00
pD -10 @ 0xa
' malloc://512
-- stdout
--- expected
+++ actual
@@ -1,3 +1,3 @@
-            0x00000000      b801000000     mov   eax, 1
-            0x00000005      4839ca         cmp   rdx, rcx
-        ,=< 0x00000008      7f00           jg    0xa
+            0x0000000a      b801000000     mov   eax, 1
+            0x0000000f      4839ca         cmp   rdx, rcx
+        ,=< 0x00000012      7f00           jg    0x14
```

File: librz/core/cmd_descs/cmd_descs.yaml (hunk 1 above)
Comment (ret2libc) [reply librz/core/cmd_descs/cmd_descs.yaml:115]: Not needed, `echo` accepts strings with spaces similar to the shell.

File: test/db/tools/rz
Comment (ret2libc) [inline test/db/tools/rz:1]: i'm not sure why this whole file has changed. I need to check

Comment (wargio) [reply test/db/tools/rz:1]: i had the same issue. is due the bytes in there that do not follow utf8.

Comment (wargio) [reply test/db/tools/rz:1]: To avoid fixing this, i suggest to extract the test `prompt settings` and make it a special file only for itself.

**Review by XVilka — CHANGES_REQUESTED** [review]
The offset bug still remains, the rest looks good.

**Review by XVilka — APPROVED** [review]
LGTM. Please also cherry-pick to https://github.com/rizinorg/rizin/pull/3847 once merged

**Pull request merged**